examples/java/veeam-vm-backup/README.MD

```diff
@@ -0,0 +1,159 @@
+
+
+## Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/java/veeam-vm-backup
+git checkout master
+```
+
+## Customize the function
+
+For security reasons to not expose sensitive data we will create a Kubernetes secret (https://kubernetes.io/docs/concepts/configuration/secret/) which will hold Veeam credentials and information, as well as the vCenter ip. This secret will be mounted into the function during runtime. This is all taken care of for your by the appliance. We only have to create the secret with a simple command through `faas-cli`.
+
+First, change the configuration file `veeamconfig.toml` holding your secret vCenter information located in the veeam-vm-backup example folder:
+
+```toml
+# veeamconfig.toml contents
+# replace with your own values
+[enterprise_manager]
+ip = "VEEAM_ENTERPRISE_MANAGER_IP"
+port = "9398"	# This is the default port for Veeam's Restful API. Change it, if you changed the port during Veeam configuration. 
+user = "USERNAME"
+password = "PASSWORD"
+
+[backup_server]
+id = "BACKUP_SERVER_ID"  # Can be found through the request to your Veeam server described here: https://helpcenter.veeam.com/docs/backup/rest/get_backupservers.html?ver=100
+
+[vcenter]
+ip = "VC_IP"
+```
+
+Now go ahead and store this configuration file as secret in the appliance.
+
+```bash
+# set up faas-cli for first use
+export OPENFAAS_URL=https://VEBA_FQDN_OR_IP
+faas-cli login -p VEBA_OPENFAAS_PASSWORD --tls-no-verify # vCenter Event Broker Appliance is configured with authentication, pass in the password used during the vCenter Event Broker Appliance deployment process
+
+# now create the secret
+faas-cli secret create veeamconfig --from-file=veeamconfig.toml --tls-no-verify
+```
+
+**Note:** Delete the local `veeamconfig.toml` after you're done with this exercise to not expose this sensitive information.
+
+Lastly, define the function-specific settings. They are performed in the `veeam-vm-backup.yml` file. Open and edit the `veeam-vm-backup.yml` provided with in the Java veeam-vm-backup example code. Change `gateway` and (optionally) `topic` as per your environment/needs.
+
+```yaml
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP # replace with your vCenter Event Broker Appliance environment
+functions:
+  veeam-vm-backup:
+    lang: java8 # or java11 for the java11 openfaas template
+    handler: ./vm-backup
+    image: vmware/veba-java-veeam-vm-backup:latest
+    secrets:
+      - veeamconfig # leave as is unless you changed the name during the creation of the secrets above
+    annotations:
+      topic: VmPoweredOffEvent  # the event can be changed, in case the aim is to back up a VM when another type of event occurs
+```
+
+## Supported Java versions
```

Comment:
This section is only relevant for developers/if someone wants to build the image vs just deploying it. Can you please break these sections up (e.g. deploy/develop) so it is not confusing for the reader which likely just wants to deploy the (pre-built) image.

Reply:
Done!